ion/symbols_token_test.go

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package ion
+
+import "fmt"
+
+// Make sure SymbolToken conforms to Stringer
+var _ fmt.Stringer = SymbolToken{}
```

Comment:
Not quite testing all of the functionality =)

Reply:
That is very fair, adding a tests for `String()`.